relay-event-normalization/src/transactions/processor.rs

```diff
@@ -2181,7 +1860,10 @@ mod tests {
             scrub_identifiers(&mut s).unwrap();
             s.0.unwrap()
         });
-        assert_debug_snapshot!(replaced);
+        assert_eq!(
+            replaced,
+            ["/*", "/*", "/*", "/test/*/url",].map(str::to_owned)
+        )
```

Comment:
This test is not really required to be updated. I decided to include it in this PR anyway, as it's the only last test remaining with a snapshot and it's very simple.